engine/flink/kafka-util/src/main/scala/pl/touk/nussknacker/engine/kafka/consumerrecord/DeserializedConsumerRecord.scala

```diff
@@ -0,0 +1,3 @@
+package pl.touk.nussknacker.engine.kafka.consumerrecord
+
+case class DeserializedConsumerRecord[K,V](value: V, key: Option[K], topic: String, partition: Int, offset: Long, timestamp: Long, headers: Map[String, Option[String]])
```

Comment:
Maybe let's `key` be before `value`? Why `key` is optional? Can you write a notice why we have own type instead of using `ConsumerRecord`?

Reply:
Fixed. Used `org.apache.kafka.clients.consumer.ConsumerRecord[K, V]` instead of `DeserializedConsumerRecord[K, V]`.